docs/content/latest/benchmark/tpcc-ysql/_index.md

```diff
@@ -63,7 +73,18 @@ Other options like username, password, port, etc. can be changed using the confi
 <password></password>
 ```
 
-## 3. Run the TPC-C benchmark
+## 3. Best Practices
+
+<b>Latest TPCC Code</b><br/>
+Use the latest enhancements of the Yugabyte TPCC application `git clone`.
+
+<b>Pre-Compacting Tables</b><br/>
+Pre-compact tables using the compact_table command in the yb-admin utility. 
+
+<b>Warming the Database</b><br/>
+Make use of the --warmup-time-secs flag when calling the execute phase of the TPCC benchmark.
```

Comment:
Run-in headings are more appropriate here. Also, what do you mean in the "latest tpcc code" item?
```suggestion
**Latest TPCC code:** Use the latest enhancements to the Yugabyte TPCC application `git clone`.

**Pre-compacting tables:** Pre-compact tables with the yb-admin utility's `compact_table` command.

**Warming the database:** Use the `--warmup-time-secs` flag when you call the execute phase of the TPCC benchmark.
```

Reply:
As opposed to downloading a specific release of the tpcc application - use git clone and get all code updates

Reply:
Ah ok, I see what you mean. I'll make a quick update and I think we'll be good to go.